src/client/css/surveyBanner.css

```diff
@@ -0,0 +1,73 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ .recruitment-banner {
+    text-align: center;
+    position: relative;
+    padding: var(--padding-md) var(--padding-md) 1.5rem;
+    background-color: var(--purple-50);
+}
+
+.recruitment-banner ul {
```

Comment:
Unrelated this this specific line, but noticed that indenting is a bit weird in this file. sometimes indented by 2, others by 4, I think line 5 has a leading space before `.recruitment-banner`. Not sure if CSS should be prettier-ed.

Reply:
Space removed in ad11567d72b884d0a90d9cd5b9a365d29d1569cb